infracost-usage-example.yml

```diff
@@ -123,7 +123,9 @@ resource_usage:
     backup_storage_gb: 10000      # Amount of backup storage that is in excess of 100% of the storage size for the cluster in GB.
 
   aws_dx_connection.my_dx_connection:
-    monthly_outbound_region_to_dx_location_gb: 100 # Monthly outbound data transferred from AWS region to DX location in GB.
```

Comment:
I wonder if we should leave the old usage param in place as well?  The two reasons I can think of are 1) backward compatability since someone may already be using it, and 2) it might be a little nicer to be able to set the "normal" non-dx gateway traffic without having to explicity mention the region.  @alikhajeh1 any thoughts?

Reply:
I think we should keep it to solve backwards compatibility. I have two questions though:

1. do we want to mark this as **deprecated** in the yaml comment in favour of the new approach
2. do we want the new approach to override the old one if the user specifies both? Or do we want to keep both working in tandem?

Reply:
Great idea with `# DEPRECATED`! I think it's fine for the new approach to override the old one so we encourage/nudge people to move towards the new way.